codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -156,14 +175,10 @@ public void sendFlowControlled(ChannelHandlerContext ctx, Http2Stream stream,
         }
         // Save the context. We'll use this later when we write pending bytes.
         this.ctx = ctx;
-        try {
-            FlowState state = state(stream);
-            state.newFrame(payload);
-            state.writeBytes(state.writableWindow());
-            flush();
-        } catch (Throwable e) {
-            payload.error(e);
-        }
+        FlowState state = state(stream);
+        state.newFrame(payload);
+        state.writeBytes(state.writableWindow());
+        flush();
```

Comment:
@buchgr - Thanks for pointing this out. I think this is OK if the exception originates from the `writeBytes` call, but do we still need a higher level catch incase the other calls throw some sort of exception?